yt_dlp/extractor/afreecatv.py

```diff
@@ -1,3 +1,4 @@
+import datetime
```

Comment:
to go with sepro's suggestion below
```suggestion
import datetime as dt
```